securityfirst/Umbrella_android
# Hotfix/checklist

CheckList release

Hot to test:
1.  Add an item, and try to remove;
2. Add an item and check the progress bar;
3. Disable an item and check the progress bar;
4. Tick items randomly;
5. Check if the percentage is correct in "My CheckList" view
